test/verify/check-storage-resize

```diff
@@ -102,6 +102,7 @@ class TestStorage(StorageCase):
                          can_grow=True, grow_needs_unmount=False)
 
     @skipImage("No NTFS support installed", "centos-7", "rhel-7-5", "rhel-7-4", "rhel-7")
+    @skipImage("ntfsck broken", "fedora-26")
```

Comment:
Hmm, I doubt it is this simple, unfortunately.

The code calls `lvresize -r`, but it should never do that for NTFS.  I'll check more closely.